pilot/pkg/bootstrap/server.go

```diff
@@ -125,6 +130,7 @@ type Server struct {
 	certController   *chiron.WebhookController
 	ca               *ca.IstioCA
 	// path to the caBundle that signs the DNS certs. This should be agnostic to provider.
+	// TODO: Unify this path with TLSOptions in PilotArgs.
```

Comment:
not sure what you mean?

Reply:
I meant PilotArgs's TLSOptions provides a way to supply cacert - I meant to combine both so that there are no two ways to specify the same thing